src/drafts/MarkdownEditor/MarkdownEditor.features.stories.tsx

```diff
@@ -0,0 +1,437 @@
+import {DiffIcon, PlusIcon} from '@primer/octicons-react'
+import React, {Meta} from '@storybook/react'
+import {useRef, useState} from 'react'
+import BaseStyles from '../../BaseStyles'
+import Box from '../../Box'
+import MarkdownEditor, {Emoji, Mentionable, Reference, SavedReply} from '.'
+import ThemeProvider from '../../ThemeProvider'
+
+const meta: Meta = {
+  title: 'Drafts/Components/MarkdownEditor/Features',
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Box sx={{maxWidth: 800}}>{Story()}</Box>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
```

Comment:
```suggestion
```

Since we wrapped each story with `<Box sx={{maxWidth:800}}>..</Box>` we don't need the decorators and its imports anymore.